evidence/pool.go

```diff
@@ -21,8 +22,9 @@ import (
 )
 
 const (
-	baseKeyCommitted = byte(0x00)
-	baseKeyPending   = byte(0x01)
+	// prefixes are unique across all tm db's
+	prefixCommitted = byte(0x08)
```

Comment:
Who's guarantying uniqueness?

Reply:
I did. I just made sure that all the prefixes across evidence, state and block db's are unique so that someone could technically have the same db for all of these without accidentally overwriting their data. It's not that important. There was however a bug in the tests because of this that took me a while to work out.

Reply:
is there anything we can do to verify they are unique? some sort of test or a script?

Reply:
You mean to stop someone from changing them in the future or introducing a new db in Tendermint which could have a prefix that matches one of these existing ones?

Reply:
> You mean to stop someone from changing them in the future or introducing a new db in Tendermint which could have a prefix that matches one of these existing ones?

yes

Reply:
Should all dB prefixes be moved to one location? Not sure if it will create cyclical dips but is a good way to monitor uniqueness